smtlib-backends.cabal

```diff
@@ -28,32 +28,10 @@ library
   ghc-options:      -Wall -Wunused-packages
   exposed-modules:
     SMTLIB.Backends
-    SMTLIB.Backends.Process
```

Comment:
But now the user has to add both `smtlib-backends` and `smtlib-backends-process` as dependencies, don't they ?

Reply:
Correct. 

You also could use [this cabal feature](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-library-reexported-modules) to expose `SMTLIB.Backends` in `smtlib-backends-process`. But I'm not sure if it is worth the trouble.

Reply:
I tried adding
```
reexported-modules: smtlib-backends-process:SMTLIB.Backends.Process
```
in `smtlib-backends.cabal` but somehow I get
```
Resolving dependencies...
Error:
    Problem with module re-exports:
      - The package 'smtlib-backends-process'
        does not export a module 'SMTLIB.Backends.Process'
    In the stanza 'library'
    In the inplace package 'smtlib-backends-0.1'
```

So unless I made an obvious mistake, yeah it's probably not worth the trouble ...

Reply:
You are trying to expose a module of `smtlib-backends-process` in `smtlib-backends`. I think `reexpoted-modules` allow you to do the converse: exposing `SMTLIB.Backends` in `smtlib-backends-process`.